src/components/VoteCell.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+export default function VoteCell(props) {
+  const { index, person, vote } = props;
+  const { id, name, voteCount } = person;
```

Comment:
이것도 너무 좋았어요~~
그런데 이제 `person`에서만 뽑아오면 되겠죠?ㅎㅎ

```suggestion
  const { id, name, voteCount } = person;
```